msi/release.sh

```diff
@@ -87,6 +90,6 @@ wix/light.exe \
 	obj/GitProduct.wixobj \
 	obj/GitComponents.wixobj \
 	-o $TARGET -ext WixUtilExtension \
-	-b / -b ../installer -sval &&
+	-b / -b ../installer &&
```

Comment:
Would you mind explaining this change in the commit message? I always stumble upon this, and I always wish the commit message was not so parsimonious with extra information that really, really helps reviewing.